src/Microsoft.Android.Build.BaseTasks/AndroidAsyncTask.cs

```diff
@@ -0,0 +1,54 @@
+using System;
+using Xamarin.Build;
+using static System.Threading.Tasks.TaskExtensions;
+
+namespace Microsoft.Android.Build.Tasks
+{
+	public abstract class AndroidAsyncTask : AsyncTask
+	{
+		/// <summary>
+		/// A helper for non-async overrides of RunTaskAsync, etc.
+		/// </summary>
+		public static readonly System.Threading.Tasks.Task Done =
+			System.Threading.Tasks.Task.CompletedTask;
```

Comment:
I think we can just remove this member, and callers can use `System.Threading.Tasks.Task.CompletedTask` directly.

Reply:
I removed this, it looks like it's only used in one place in xamarin-android and is easy enough to replace.